tests/test_mapper.py

```diff
@@ -279,3 +281,73 @@ class Employee:
     assert type(name.type) == StrawberryOptional
     id = next(iter(filter(lambda f: f.name == "department", employee_type_fields)))
     assert type(id.type) == StrawberryOptional
+
+
+def test_relationships_schema(employee_and_department_tables, mapper):
+    EmployeeModel, DepartmentModel = employee_and_department_tables
+
+    @mapper.type(EmployeeModel)
+    class Employee:
+        __exclude__ = ["password_hash"]
+
+    @mapper.type(DepartmentModel)
+    class Department:
+        pass
+
+    @strawberry.type
+    class Query:
+        @strawberry.field
+        def departments(self) -> Department:
+            ...
+
+    mapper.finalize()
+    schema = strawberry.Schema(query=Query)
```

Comment:
Without this change this line by itself would error out.
The expected sql below is just to have something extra to assert for